ansible_collections/arista/avd/roles/eos_cli_config_gen/README.md

```diff
@@ -1587,6 +1588,21 @@ snmp_server:
       enable: < true | false >
 ```
 
+###  System Control-Plane
+```yaml
+system:
+  control_plane:
+    tcp_mss:
+      ipv4: < Segment size >
+      ipv6: < Segment size >
+    ipv4_access_group:
```

Comment:
I missed this the last time. We should use plural here, so `ipv4_access_groups`.

Reply:
Fixed in 7f64921e53cc2146c20a104a9fb2105c2ad1d79d